### ISSUE TITLE
timeout fix

### ISSUE THREAD

**Issue by AlexZeDim** [lukemnet/blizzapi#224]
I have found a little blind spot in the `query.ts` code, relevant to the `timeout` param.

For somehow, the `timeout` param was ignored by some queries. So even, if it was predetermined, it gives an error with the default timeout.

```
  let market = await api.query(`/data/wow/connected-realm/${_id.connected_realm_id}/auctions`, {
    timeout: 30000,
    params: { locale: 'en_GB' },
    headers: {
      'Battlenet-Namespace': 'dynamic-eu',
      'If-Modified-Since': _id.timestamp
    }
  }).catch(error => {
    console.info(`E,${_id.connected_realm_id}:${error}`);
    return void 0;
  });
```

![timeout10s](https://user-images.githubusercontent.com/907696/96040575-38172180-0e73-11eb-9be2-f3fee4299f36.png)

So, I have made a PR, and this time, all Travis and Codecov checks are green-lighting, so as I guess, it won't a problem to add it.

**Comment by lwojcik** [comment]
@AlexZeDim Thanks for the reporting! Good call, I feel a little ashamed I didn't notice this while implementing the change.

**Comment by lwojcik** [comment]
Merged, added missing tests and released in [v1.3.3](https://github.com/lukemnet/blizzapi/releases/tag/v1.3.3).
